cypress/integration/plugins/security-dashboards-plugin/oidc_test.spec.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Log in via OIDC', () => {
+  const login = 'admin';
+  const password = 'admin';
+
+  const kcLogin = () => {
+    cy.get('#kc-page-title').should('be.visible');
+    cy.get('#username').type(login);
+    cy.get('#password').type(password);
+    cy.get('#kc-login').click();
+  };
+
+  const logout = () => {
+    cy.get('#user-icon-btn').should('be.visible', { timeout: 15000 });
+    cy.get('#user-icon-btn').click();
+    cy.get('button[data-test-subj^="log-out-"]').click();
+    cy.get('#kc-page-title').should('be.visible');
+  };
+
+  afterEach(async () => {
```

Comment:
Should we also include `cy.clearCookies()` here?

Reply:
@jakubp-eliatra WDYT?

Reply:
> @sebastianmichalski Took a first pass and overall this looks very good. Thank you for the contribution!
> 
> I see this PR adds a yarn.lock file when there is already a package-lock.json. Should there only be a single lockfile?

Correct, will fix that as well. :)